packages/graph/hash_graph/tests/integration/postgres/property_type.rs

```diff
@@ -33,7 +33,7 @@ fn query() {
         .expect("could not create property type");
 
     let property_type = database
-        .get_property_type(created_property_type.version_id())
+        .get_property_type(*created_property_type.version_id())
```

Comment:
nit:
These derefs are because of the changes to the getters earlier returning references, right? Could we change the database helpers to take refs as well, or does it just not matter? I don't have strong feelings about this, but just a little overhead when reading the code

Reply:
This is in the tests, so I don't know if we would benefit from it. The public API is using the idiomatic way to not pass a `Copy`-value by reference. I don't mind either way.